calc.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+
+def add(a, b):
```

Comment:
The number of commits should be at least as number of functions in the app (as I wrote in the 1st PR "Separate each logical change into a separate commit"). Each commit should have:
1. Logic + test + README part.
2. https://www.kernel.org/doc/html/v4.10/process/submitting-patches.html - describe the commit + save the 
3. Your message write is equal to reading the diff you don't describe it.  I want to see commit such us:

calc: new command 'add' 
- describe the logic
- Added sanity test to verify the base logic works
- Update README

Reply:
Split also the main logic and the general logic as the last commit